sample/Tailor/tools/bazelwrapper

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```

Comment:
we should have a single bazelwrapper for the repo rather than one per sample project IMO

Reply:
I agree, we probably need to address assumptions in that shell script regarding the PWD in order to pull that off. Additionally, we may need to have a bazelwrapper that parameterizes the version